src/open_source_python_template/crawlTasks.py

```diff
@@ -18,7 +18,9 @@ def get_tasks():
         if creds and creds.expired and creds.refresh_token:
             creds.refresh(Request())
         else:
-            flow = InstalledAppFlow.from_client_secrets_file("credential.json", SCOPES)
+            flow = InstalledAppFlow.from_client_secrets_file(
+                "src/open_source_python_template/credential.json", SCOPES
```

Comment:
is there a reason why we have the entire path as opposed to the relative path here? The import didn't work for me with the full path

Reply:
That‘s kinda wired because when I do not use the entire path it doesn't work for me.

Reply:
resolved, thanks!